save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/Utils.kt

```diff
@@ -33,11 +34,15 @@ import kotlinx.serialization.json.Json
  * @return a nicely formatted string representation of [FileInfo]
  */
 @Suppress("MAGIC_NUMBER", "MagicNumber")
-fun FileInfo.toPrettyString() = "${key.name} (uploaded at ${
-    Instant.fromEpochMilliseconds(key.uploadedMillis).toLocalDateTime(
-        TimeZone.UTC
-    )
-}, size ${sizeBytes / 1024} KiB)"
+fun AbstractFileInfo.toPrettyString() = if (this is FileInfo) {
```

Comment:
suggestion to move it to props in FileUploader as method

Reply:
Makes sense

Reply:
Done